plugin/index.ts

```diff
@@ -156,8 +155,8 @@ export const visualizer = (
 
       const filter = createFilter(opts.include, opts.exclude);
 
-      const gzipSize = !!opts.gzipSize && !opts.sourcemap;
-      const brotliSize = !!opts.brotliSize && !opts.sourcemap;
+      const gzipSize = !!opts.gzipSize;
+      const brotliSize = !!opts.brotliSize;
```

Comment:
I would print a warning also. It is not good UX to just silently disable options.